static/app/utils/replayCount/useReplayCount.tsx

```diff
@@ -62,7 +62,10 @@ export default function useReplayCount({
             data_source: dataSource,
             project: -1,
             statsPeriod,
-            query: `${fieldName}:[${ids.join(',')}]`,
+            query:
+              fieldName === 'transaction'
```

Comment:
Is this only applicable to `transaction`? Would it be safe(r) to quote for all fields?

Reply:
<img width="547" alt="SCR-20241015-lusn" src="https://github.com/user-attachments/assets/134c4fb6-6427-4499-b0c5-bde715877750">
seems to break if we do it for all cases, since `issue.id` expects number(s) not string